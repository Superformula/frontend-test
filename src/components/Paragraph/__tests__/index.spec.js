import React from 'react';
import { mount } from 'enzyme';
import { Paragraph } from '../';

describe('Paragraph', () => {
  it('should render without failing', () => {
    const wrapper = mount(<Paragraph>Lorem ipsum</Paragraph>);
    expect(wrapper).toMatchSnapshot();
  });
});
