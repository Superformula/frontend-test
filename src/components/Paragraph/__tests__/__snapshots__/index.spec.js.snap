// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render without failing 1`] = `
.c0 {
  color: #606060;
  max-width: 792px;
  font-size: 16px;
}

@media (min-width:758px) {
  .c0 {
    font-size: 22px;
    -webkit-letter-spacing: .92px;
    -moz-letter-spacing: .92px;
    -ms-letter-spacing: .92px;
    letter-spacing: .92px;
    font-weight: 300;
  }
}

<Paragraph>
  <p
    className="c0"
  >
    Lorem ipsum
  </p>
</Paragraph>
`;
